apps/prairielearn/elements/pl-matching/pl-matching.py

```diff
@@ -116,16 +117,23 @@ def make_option(name: str, html: str) -> dict[str, int | str]:
             options[option_name] = option
 
         elif child.tag in ["pl-statement", "pl_statement"]:
-            pl.check_attribs(child, required_attribs=["match"], optional_attribs=[])
+            pl.check_attribs(
+                child, required_attribs=["match"], optional_attribs=["aria-label"]
+            )
             child_html = pl.inner_html(child)
             match_name = pl.get_string_attrib(child, "match")
+            aria_label = pl.get_string_attrib(child, "aria-label", None)
```

Comment:
Should have this default set as a constant at the top of the file.

Reply:
Fixed in [bc9b569](https://github.com/PrairieLearn/PrairieLearn/pull/11976/commits/bc9b569377b031243fe00a4f98fb1260b36ef58c).